ert_shared/ensemble_evaluator/config.py

```diff
@@ -6,7 +6,7 @@
 CLIENT_URI = "client"
 DISPATCH_URI = "dispatch"
 DEFAULT_PORT = "8765"
-DEFAULT_HOST = "localhost"
+DEFAULT_HOST = "0.0.0.0"
```

Comment:
I think this should be exceptional behavior? `0.0.0.0` is maybe a bit too open, especially without any auth.

Reply:
Agreed, this was just to do some testing and should be removed before we merge it!